tests/environment.rs

```diff
@@ -0,0 +1,72 @@
+extern crate assert_cli;
```

Comment:
Can you move all the tests into a `#[cfg(test)] mod test` in `environment.rs` and use the real `with_env`?

Reply:
The fact is, with_env doesn't allow to test the environment object after ...